packages/postcss-syntax/src/parse.ts

```diff
@@ -5,16 +5,18 @@ import { BabelPluginOptions } from '@griffel/babel-preset';
 
 export type PostCSSParserOptions = Pick<postcss.ProcessOptions<postcss.Document | postcss.Root>, 'from' | 'map'>;
 
-export interface ParserOptions extends PostCSSParserOptions, BabelPluginOptions {}
+export interface ParserOptions extends PostCSSParserOptions {
+  griffelPreset?: BabelPluginOptions;
+}
 
 export const parse = (css: string | { toString(): string }, opts?: ParserOptions) => {
-  // eslint-disable-next-line @typescript-eslint/no-unused-vars
-  const { from: filename = 'postcss-syntax.styles.ts', map, ...griffelPluginOptions } = opts ?? {};
+  const { from: filename = 'postcss-syntax.styles.ts', griffelPreset } = opts ?? {};
```

Comment:
postcss passes some untyped properties here - it will fail the griffel config validator, so I've put it all the preset configuration in a separate property